third-party/realdds/src/dds-notification-server.cpp

```diff
@@ -127,6 +126,8 @@ void dds_notification_server::send_notification( topics::flexible_msg && notific
     std::unique_lock< std::mutex > lock( _notification_send_mutex );
     if( ! _instant_notifications.enqueue( std::move( raw_notification ) ) )
         LOG_ERROR( "error while trying to enqueue a notification" );
+    _new_instant_notification = true;
```

Comment:
Who is using this?

Reply:
`_notifications_loop` active object is using this to signal that the are messages waiting to be sent that are not the init messages